packaging/docker/Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN curl -o /tmp/cloudify-manager-install.rpm $rpm_file \
     && sed -i 's/ssl_enabled: true/ssl_enabled: false/' /opt/cloudify/cfy_manager/lib/python3.6/site-packages/config.yaml \
     && rm -fr /opt/cloudify/sources/*.rpm -fr \
     && rm /etc/yum.repos.d/Cloudify-Local.repo \
-    && if [ "$autoremove" == "true" ]; then yum autoremove -y; fi \
+    && if [ "$arch" != "aarch64" ]; then yum autoremove -y; fi \
```

Comment:
unfortunate that back then we didn't comment what's the problem with autoremove. So at least now would be a good time.

Reply:
Did you push? I can't see a comment about this in this file.